.github/workflows/upload_to_docker_hub.yml

```diff
@@ -7,13 +7,13 @@ on:
       - 'v*'
```

Comment:
¿Qué pasa si actualizas el contenedor en otra rama?
De veras no entiendo por qué tenéis la manía de actualizar la imagen sólo en ramas.

Reply:
Después del Objetivo 5 voy a cambiarlo para que sólo se actualice cuando hay un push a  master. Creo que es lo más sensato.